types/pg/index.d.ts

```diff
@@ -1,8 +1,8 @@
-// Type definitions for pg 7.11
+// Type definitions for pg 7.14
 // Project: http://github.com/brianc/node-postgres
 // Definitions by: Phips Peter <https://github.com/pspeter3>, Ravi van Rooijen <https://github.com/HoldYourWaffle>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.7
```

Comment:
Why was this necessary?

Reply:
Not necessary at all I believe. Changed back.